src/app/services/project_service.js

```diff
@@ -221,10 +223,32 @@ angular
                 return _.includes(['model', 'source', 'seed', 'snapshot', 'analysis', 'exposure'], node.resource_type);
             });
 
-            service.project.searchable = _.filter(search_nodes.concat(search_macros), function(obj) {
+            service.project.searchable = prepareModelsForSearching(_.filter(search_nodes.concat(search_macros), function(obj) {
                 // It should not be possible to search for hidden documentation
                 return !obj.docs || obj.docs.show;
-            });
+            }));
+
+			const fuseOptions = {
+			  includeScore: true,
+			  includeMatches: true,
+			  ignoreLocation: true,
```

Comment:
My understanding of [Fuse's options](https://fusejs.io/api/options.html#fuzzy-matching-options) is that by setting ignoreLocation, there's no point in setting threshold. Does that sound right to you? 

I'd like to tone down the fuzziness a bit, but it's not in terms of how close to the start of the string a term is, it's how aggressively it thinks character replacements/transpositions should be to get a match. And I don't think it offers any other levers to pull.

Reply:
hm hm hm! I am reading this slightly differently - it sounds so me like `ignoreLocation` will score matches in _any_ part of the string, but I believe that _threshold_ still has a role to play! I played around with this by setting threshold to `0.1`, then searching for "zz" and did not see any matches. Next, I changed the threshold to `0.9` and searched for "zz" and I _did_ see matches!

I bet we can make the search less fuzzy by reducing the threshold from the default (`0.6`) to a smaller number... unclear to me how to pick that number beyond guess-and-check'ing though :)

Reply:
You're right! Currently on 0.4 - I tried 0.3 but that excluded some things I thought should be allowed.